### DISCUSSION TITLE
Re-add the show_nitf++ build target

### DISCUSSION THREAD

**Discussion by thompsonab** [mdaus/nitro#438] [Ideas]
Version: v2.10.7 (or current master, commit 9ea4474)
Build system: CMake
Topic of issue: Re-add the `show_nitf++` build target

The previous Waf-based build system defined a `show_nitf++` target that was very useful for dumping the known metadata within a given NITF. It was also a great way to demonstrate how to actually use the NITRO API to extract said metadata programmatically.

In the process of migrating from Waf to CMake, the `show_nitf++` app was not included as a build target. I would like to propose that it be re-added as a valid build target. It seems that the changes would be relatively minor/noninvasive, merely adding two lines to add the executable target and link it to the appropriate library.

I would be happy to submit a PR once I receive approval from the appropriate people on my project.

**Comment by thompsonab** [comment]
@JDanielSmith I received the go-ahead from my project's engineering lead to proceed with the corresponding PR if it's something you think would be of interest or value

**Reply by JDanielSmith** [reply]
That would be great!  Hook it up and I'll merge.

**Reply by thompsonab** [reply]
PR submitted. Thanks!
